feature-libs/checkout/base/components/checkout-order-confirmation/checkout-order-confirmation.module.ts

```diff
@@ -53,11 +60,21 @@ const orderConfirmationComponents = [
           guards: [OrderConfirmationGuard],
         },
         OrderConfirmationOverviewComponent: {
-          component: CheckoutOrderConfirmationOverviewComponent,
+          component: OrderDetailShippingComponent,
```

Comment:
we missed this from cart-lib way back from our checkout-cart-merge

I deleted the component from out lib and used the same way they did on re-using the 'same' component found in order lib

- doing so I had to change the `.getOrder()` method from the checkoutfacade to `.getOrderDetails`. Reason being, the rest of the codebase uses .getOrderDetails instead of getOrder to reference the order. Most of the ones in the codebase uses these DI, so I had to match the method definition as well.